addons/mail/static/src/core/common/message.js

```diff
@@ -411,12 +411,19 @@ export class Message extends Component {
         if (ev.target.tagName === "A") {
             if (model && id) {
                 ev.preventDefault();
-                this.env.services.action.doAction({
+                await this.env.services.action.doAction({
```

Comment:
Are you sure it's still necessary to add this extra `await` when manually inserting the chat window?

I would expect it to show as soon as the action is executed if an actual chat window is opened (rather than thread open, which can open in discuss as discussed before).